ansible/templates/telegraf_balance.conf.j2

```diff
@@ -0,0 +1,12 @@
+[[inputs.exec]]
```

Comment:
this should go in the https://github.com/eoliveros/zap_dashboard repo.. it is specific to the a zapd install and not related to xchwallet